consensus/obcpbft/pbft-core_mock_test.go

```diff
@@ -47,7 +47,7 @@ func (pe *pbftEndpoint) isBusy() bool {
 	// channel, the send blocks until the thread has picked up the new work, still
 	// this will be removed pending the transition to an externally driven state machine
 	select {
-	case <-pe.pbft.idleChan:
+	case pe.pbft.manager.queue() <- nil:
```

Comment:
This won't work with my previous loop suggestion;   however, this code would change anyways later.